target/linux/mediatek/patches-6.1/251-pinctrl-mediatek-mt7981-add-additional-emmc-group.patch

```diff
@@ -0,0 +1,56 @@
+mediatek: mt7981: pinctrl: add additional emmc groups
```

Comment:
Has this been sent upstream?
If so, please add `X-Patchwork-*` headers (along with other RFC822 headers which would be needed to re-send the patch upstream)

Reply:
Yes, the patch has been merged upstream in linusw tree (https://git.kernel.org/pub/scm/linux/kernel/git/linusw/linux-pinctrl.git/commit/?h=for-next&id=398f914bf0022dcde82050aab72a2bd842ee1709). I've taken the patch from https://patchwork.kernel.org in order to add the requested headers, please let me know if any are missing.